src/os_esp32.cc

```diff
@@ -550,6 +552,7 @@ class HeapSummaryPage {
     memset(void_cast(counts_), 0, sizeof counts_);
     users_ = 0;
```

Comment:
Why not initialize these at the declaration sites?

Reply:
Page objects get reused when we increment dropped_pages_ so this code is not only run in the constructor.